src/ServiceControl/Operations/FailedAuditImportCustomCheck.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace ServiceControl.Operations
+{
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using Raven.Client;
+    using Raven.Client.Indexes;
+    using System;
+    using System.Linq;
+
+    class FailedAuditImportCustomCheck : CustomCheck
+    {
+        public FailedAuditImportCustomCheck(IDocumentStore store)
+            : base("Audit Message Ingestion", "ServiceControl Health", TimeSpan.FromHours(1))
+        {
+            this.store = store;
+        }
+
+        public override CheckResult PerformCheck()
+        {
+            using (var session = store.OpenSession())
+            {
+                var query = session.Query<FailedAuditImport, FailedAuditImportIndex>();
+                using (var ie = session.Advanced.Stream(query))
+                {
+                    if (ie.MoveNext())
+                    {
+                        var message = @"One or more audit messages have previously failed to import properly into ServiceControl and have been stored in the ServiceControl database.
+Due to a defect however, ServiceControl would not be able to automatically reimport them. Please run ServiceControl in the maintenance mode and use embedded RavenStudio available by default at http://localhost:33333/storage to examine the payloads of failed messages to ensure no information has been lost.
```

Comment:
Can we at least mention the document collection that someone would look for these in?

Reply:
These messages will all change once we have the workaround in place. These are currently just copy-pasted from the original log message. Once we have the re-import we can then tell the customers about that in the message. The re-import can have a more detailed "we've tried to reimport these records and it was a bust. Open a support case"

Reply:
Well, if the messages will be present before we implement the work-around we can still provide more info on where to look for these messages can't we?

Reply:
I'll mark this as WIP until the command line stuff is ready then.

Reply:
I've just merged the command line PR.